packages/file/jest.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
```

Comment:
Might be nice to be consistent and use the exact same config as in https://github.com/sketch-hq/sketch-document/blob/a0464fd5b29b29c22dede03a31219c56ef62a3fa/packages/file-format-ts/jest.config.js#L1-L5

```suggestion
  testEnvironment: 'node',
  testRegex: '^.*\\.test.ts$',
```